docker/simnet.yml

```diff
@@ -149,7 +149,8 @@ services:
       context: ../
       dockerfile: docker/lspd/Dockerfile
     environment:
-      - NODE_PUBKEY
+      NODE_HOST: 10.5.0.3:9736
```

Comment:
I think it should be only 10.5.0.3 without the port as it uses the default port.
In different than dev-simnet.yml where there is port mapping to NODE_HOST, here we are using the internal network IP address so no port mapping is done between 9735 to 9736.